azure-pipelines.yml

```diff
@@ -9,6 +9,9 @@ jobs:
         OS_DISTRO: ubuntu
         PUSH_GCR_IMAGE: true
         GCR_IMAGE_NAME: envoy-build
+      mobile:
```

Comment:
i think you shouldnt need this anymore

Reply:
Ah yes, thanks for pointing that out.

Reply:
~relatedly we might want to add a `NO_BUILD_VARIANTS` or similar